features/4-Events/Create/email-notification.feature

```diff
@@ -0,0 +1,47 @@
+# feature title should be more specific
+
+Feature: Notification when new parties are created
+
+
+  In order to keep track of new parties and their details
+
+  As an admin
+
+  I want to get an email notification from the application when a party is created by a host
+
+
+# Scenarios usually describe different paths through the feature, so there
+
+# would be various scenarios, e.g.
+
+
+Scenario: Host creates new party
+
+...
+
+...
+
+...
+
+# Admin should not be sent an email
+
+Scenario: Admin creates new party
+
```

Comment:
This scenario should be added.  It is to record the fact that we don't send out an email notification if an administrator has created the party.